tests/odov3/odo_test.go

```diff
@@ -61,30 +55,28 @@ func TestOdo(t *testing.T) {
 	// more at: https://onsi.github.io/ginkgo/#dynamically-generating-specs
 	g := NewGomegaWithT(t)
 
-	// make stacks path absolute, it is required because later we are using it from the random directory.
-	stacksDir, err := filepath.Abs(stacksDir)
-	g.Expect(err).To(BeNil())
+	files := make([]string, 0)
```

Comment:
That is not possible because `var files []string` errors out because its not an iterator. That was originally was I was going to do. Instead we need to instantiate an array or slice with length 0.